src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -737,4 +737,18 @@ function ConvertFrom-SddlString
             RawDescriptor = $rawSecurityDescriptor
         }
     }
-}
\ No newline at end of file
+}
+
+function Get-Uptime
+{
+    [CmdletBinding()]
+    [Alias("uptime")]
+    [OutputType([System.Timespan])]
+    Param()
+
+    Process
+    {
+        # It is portable
+        [TimeSpan]::FromSeconds([system.diagnostics.stopwatch]::GetTimestamp()/[system.diagnostics.stopwatch]::Frequency)

```

Comment:
@JamesWTruher    Is there universal (portable) way to calculate uptime?

I agree with "if IsHighResolution" but I do not like "throw". Really today are there systems where it is possible? Perhaps a better return $null?